doc/doxygen/example_particles.dox

```diff
@@ -130,7 +130,8 @@
 
 /** \example particles/particles2.c
  *
- * 2D version of a generic particle tracking demo.
+ * 2D version of a generic particle tracking demo (3D counterpart:
```

Comment:
The first line shows up as a short title in the list on the web page. The information in parentheses is well meant, but maybe rather put into the text, not the first line, instead?